libs/web/components/src/IdentityFormDialog.tsx

```diff
@@ -29,8 +29,11 @@ export const IdentityFormDialog: FC<IdentityFormDialogProps> = ({
 				<Choose>
 					<Choose.When condition={formState?.status === "Ok"}>
 						<p className="prose text-center">
-							Your identity has been successfully set. Visit Discord to view the
-							Governance channels with your new role!
+							Your identity has been successfully set.&nbsp;
+							<If condition={!!formState?.cennzDiscordMember}>
+								Visit Discord to view the Governance channels with your new
+								role!
+							</If>
```

Comment:
Should keep the message but just omit the `"with your new role!"`

Reply:
I believe they won't have access to the channels without the role

Reply:
Okay, then can tweak the message and send user to the generic channel instead? Feels like we remove the message for this scenario but will offer a CTA to go to Discord